MODULE.bazel

```diff
@@ -0,0 +1,83 @@
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.4.2")
+bazel_dep(name = "aspect_rules_js", version = "1.32.6")
+
+npm = use_extension("@aspect_rules_js//npm:extensions.bzl", "npm")
+npm.npm_translate_lock(
+    name = "aspect_rules_format_npm",
+    link_workspace = "aspect_rules_format",
+    pnpm_lock = "//:pnpm-lock.yaml",
+)
+use_repo(npm, "aspect_rules_format_npm")
+
+bazel_dep(name = "buildifier_prebuilt", version = "6.3.3")
+bazel_dep(name = "aspect_bazel_lib", version = "1.11.1")
+bazel_dep(name = "rules_python", version = "0.26.0")
+bazel_dep(name = "rules_java", version = "6.4.0")
+bazel_dep(name = "rules_go", version = "0.39.1", repo_name = "io_bazel_rules_go")
+
+go = use_extension("@io_bazel_rules_go//go:extensions.bzl", "go_sdk")
+go.download(
+    name = "go_sdk",
+    version = "1.19.1",
+)
+use_repo(go, "go_sdk")
+
+non_module_deps = use_extension("//:non_module_deps.bzl", "non_module_deps")
+use_repo(non_module_deps, "scala_reflect")
+use_repo(non_module_deps, "scalafmt_cli_2_13")
+use_repo(non_module_deps, "diffutils")
+use_repo(non_module_deps, "scalafmt_config_2_13")
+use_repo(non_module_deps, "paiges_core_2_13")
+use_repo(non_module_deps, "scalameta_2_13")
+use_repo(non_module_deps, "jline")
+use_repo(non_module_deps, "sourcecode_2_13")
+use_repo(non_module_deps, "scalap")
+use_repo(non_module_deps, "scalafmt_interfaces")
+use_repo(non_module_deps, "scala_compiler")
+use_repo(non_module_deps, "scala_parallel_collections_2_13")
+use_repo(non_module_deps, "java_diff_utils")
+use_repo(non_module_deps, "fansi_2_13")
+use_repo(non_module_deps, "trees_2_13")
+use_repo(non_module_deps, "parsers_2_13")
+use_repo(non_module_deps, "jna")
+use_repo(non_module_deps, "config")
+use_repo(non_module_deps, "scalafmt_core_2_13")
+use_repo(non_module_deps, "scopt_2_13")
+use_repo(non_module_deps, "interface")
+use_repo(non_module_deps, "scalafmt_sysops_2_13")
+use_repo(non_module_deps, "metaconfig_pprint_2_13")
+use_repo(non_module_deps, "svm_subs")
+use_repo(non_module_deps, "metaconfig_typesafe_config_2_13")
+use_repo(non_module_deps, "common_2_13")
+use_repo(non_module_deps, "scala_collection_compat_2_13")
+use_repo(non_module_deps, "metaconfig_core_2_13")
+use_repo(non_module_deps, "scala_library")
+use_repo(non_module_deps, "scalafmt_dynamic_2_13")
+use_repo(non_module_deps, "nailgun_server")
+use_repo(non_module_deps, "buf")
+use_repo(non_module_deps, "terraform_linux_x86_64")
+use_repo(non_module_deps, "swiftformat")
+use_repo(non_module_deps, "jsonnet_linux_x86_64")
+use_repo(non_module_deps, "google-java-format")
+use_repo(non_module_deps, "ktfmt")
+
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
+python.toolchain(
+    is_default = True,
+    python_version = "3.11",
+)
+
+pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
+pip.parse(
+    hub_name = "pip",
+    python_version = "3.11",
+    requirements_lock = "//:requirements_lock.txt",
```

Comment:
How does a user customise the version of black that is installed with this approach? In some cases typing_extensions also needs installing for black to work correctly.

Reply:
This repo is the formatter, and we always wanted the linter. I have started one.

In rules_lint, the answer is "we don't try to provide the toolchain or the tools, you must provide them".

For example, here's how we get `flake8`:
https://github.com/aspect-build/rules_lint/blob/main/example/lint.bzl#L18

If a user is formatting code in a certain language, they probably already have the toolchain for that language. It wouldn't be too much effort for them to add three lines to a BUILD file to declare the `black` binary for us.

This would help with the error above about Go as well. WDYT?

Reply:
Okay I'm doing some prototyping over here https://github.com/aspect-build/rules_lint/pull/14

So far it looks promising

Reply:
Okay it's more than a prototype. Pretty sure that's the right way to do this with bzlmod, and this repo can just be the WORKSPACE way. Would love your input @agluszak